.mergify.yml

```diff
@@ -1,6 +1,21 @@
 # See https://doc.mergify.io
 
 pull_request_rules:
+  - name: label core
+    actions:
+      label:
+        add: [ contribution/core ]
+    conditions:
+      - author=eladb
+      - author=RomainMuller
```

Comment:
Are you sure this works? I think the conditions are combined through an AND operation and not through OR.

Reply:
Also replace the condition below that lists all the owners with the label. We don’t need this list twice.

Reply:
OMG you're right.

And w/r/t the label... Have to be 100% certain that nobody off the list can assign labels.

Reply:
Ahh, I didnt realize this